mlflow/runs.py

```diff
@@ -0,0 +1,74 @@
+"""
+CLI for runs
+"""
+import click
+import datetime
+import json
+from mlflow.entities import ViewType
+from mlflow.tracking import _get_store
+from tabulate import tabulate
+
+RUN_ID = click.argument("run_id", type=click.STRING)
+
+
+@click.group("runs")
+def commands():
+    """
+    Manage runs. To manage runs of experiments associated with a tracking server, set the
+    MLFLOW_TRACKING_URI environment variable to the URL of the desired server.
+    """
+    pass
+
+
+@commands.command("list")
+@click.option("--experiment_id", help="Specify the experiment for list of runs.", type=click.INT)
+@click.option("--view", "-v", default="active_only",
+              help="Select view type for list experiments. Valid view types are "
+                   "'active_only' (default), 'deleted_only', and 'all'.")
+def list_run(experiment_id, view):
+    """
+    List all runs of the specified experiment in the configured tracking server.
+    """
+    store = _get_store()
+    view_type = ViewType.from_string(view) if view else ViewType.ACTIVE_ONLY
+    runs = store.list_run_infos(experiment_id, view_type)
+    table = [[run.run_uuid, run.name, datetime.datetime.fromtimestamp(run.start_time / 1000.0).strftime('%Y-%m-%d %H:%M:%S')] for run in runs]
+    print(tabulate(sorted(table), headers=["Run Id", "Name", "Start time"]))
```

Comment:
Let's call the columns "Date", "ID", and "Name" -- "Date" from the MLflow UI. I think "ID" and "Name" are reasonable since we're clearly in a runs context, but "Run ID" and "Run Name" would also be reasonable.